aten/src/ATen/native/quantized/cudnn/BinaryOps.cpp

```diff
@@ -73,18 +74,28 @@ Tensor add(Tensor qa, Tensor qb, double output_scale, int64_t output_zero_point)
     }
     qa = qa.view(new_sizes);
     qb = qb.view(new_sizes);
+  } else if (qa.dim() == 4) {
+    qa = qa.contiguous(c10::MemoryFormat::ChannelsLast);
+    qb = qb.contiguous(c10::MemoryFormat::ChannelsLast);
   }
 
-  at::Tensor add_output = at::empty(qa.sizes(), at::device(at::kCUDA).dtype(at::kFloat));
-  at::Tensor quantized_output = at::_empty_affine_quantized(
-      qa.sizes(),
-      at::device(at::kCUDA).dtype(at::ScalarType::QInt8),
-      output_scale,
-      output_zero_point);
+  at::Tensor add_output = qa.dim() == 4 ? at::empty(qa.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast)
+                                        : at::empty(qa.sizes(), at::device(at::kCUDA).dtype(at::kFloat));
```

Comment:
nit: there are some duplicated code here, can we have a variable to represent memoryformat and only apply the branch for the variable? like:
```
memory_format = a if xxx else b
add_output = at:empty(qa.sizes(), ..., memory_format)
```